README.md

```diff
@@ -454,6 +454,23 @@ To use the JAR with no dependencies in a Java application, the following require
 ### Building and using the Javadoc JAR to extract the Javadoc HTML files (amazon-timestream-jdbc-\<version\>-javadoc.jar)
 Javadoc JAR builds with `mvn install` alongside the other JAR files. To extract the Javadoc HTML files, use the following command: `jar -xvf amazon-timestream-jdbc-1.0.0-javadoc.jar`
 
+### Debug with BI tools
+#### DBeaver
+To enable debug logs, the application should be run with `-Ddbeaver.trace.enabled=true` flag
+
+#### DbVisualizer
+To enable debug mode, the following steps should be done:
+1. Open `Tools -> Debug Window`
+2. Open the `Debug Log` tab
+3. Enable the `Debug DbVisualizer` checkbox
```

Comment:
Add instructions on where to view log files, and also an example of the actual log generated by the driver.